packages/components/src/Checkbox.js

```diff
@@ -39,7 +39,7 @@ const CheckboxIcon = props => (
 
 export const Checkbox = React.forwardRef(
   ({ className, sx, variant = 'checkbox', ...props }, ref) => (
-    <Box>
+    <Box sx={{ position: 'relative' }}>
```

Comment:
Thanks! But generally we shouldn't introduce any positioning to the default styles if possible since it will create a new positioning context and can cause unintended style bugs -- can you link to a reproduction of the issue you're seeing along with a screenshot for context here?

Reply:
@jxnblk sorry for the delay. 
Here is the example https://f79d6.sse.codesandbox.io/
This is kinda related to form behavior. If you focus on an input element form will be automatically scrolled to that position.
I'll demonstrate it in a video:
https://monosnap.com/file/cCJMMlZvXtBXzIR61DO0nDCirTG0a8

Reply:
Sorry, I added `position: relative` to `label` element in the video, but it's the same fix if you add it on checkbox input wrapper `div` element.

Reply:
Based on the video, that *looks* like normal browser behavior -- not sure if I'm misunderstanding this/don't see a link to the code, but maybe you can clarify the issue?

Reply:
@jxnblk the problem is in an extra gap/area at the bottom of the screen which makes the whole body jump (and scrollable), that should not happen, instead only light blue sidebar with red border should be scrollable.
The fix is to add `position: relative` to checkbox wrapping div.